src/main/scala/coreplex/Configs.scala

```diff
@@ -15,11 +15,10 @@ import freechips.rocketchip.util._
 
 class BaseCoreplexConfig extends Config ((site, here, up) => {
   // Tile parameters
-  case PAddrBits => 32
   case PgLevels => if (site(XLen) == 64) 3 /* Sv39 */ else 2 /* Sv32 */
   case ASIdBits => 0
   case XLen => 64 // Applies to all cores
-  case ResetVectorBits => site(PAddrBits)
+  case ResetVectorBits => 32 // TODO: site(SharedMemoryTLEdge).bundle.addressBits
```

Comment:
If this proves difficult to get rid of, it's probably OK to use ```xLen``` for this, then truncate it to ```paddrBits min xLen``` in the Frontend.

Reply:
(I don't mean ```case ResetVectorBits => site(XLen)```, I mean delete the parameter altogether)

Reply:
The problem isn't so much inside the tile as it is at the coreplex/system level. Some coreplexes pop this wire out as an external signal, but the width of that signal needs to be based on how much memory the tiles can actually see. I'm not even sure what it should be if e.g. different tiles can see different amounts of memory. It's possibly that having a single global reset vector is actually a misnomer

Reply:
Agreed. I can think of lots of reasons you may want different reset vector for different cores.